activesupport/lib/active_support/json/decoding.rb

```diff
@@ -19,8 +19,9 @@ class << self
       #
       #   ActiveSupport::JSON.decode("{\"team\":\"rails\",\"players\":\"36\"}")
       #   => {"team" => "rails", "players" => "36"}
-      def decode(json)
-        data = ::JSON.parse(json, quirks_mode: true)
+      def decode(json, options = {})
+        parse_options = options.merge(quirks_mode: true)
```

Comment:
`quirks_mode` is not longer a thing since 2016: https://github.com/ruby/json/commit/7d2ad6d6556da03300a5aeadeeacaec563435773

I'll remove it on `main`